site/docs/blogs.md

```diff
@@ -23,6 +23,37 @@ title: "Blogs"
 Here is a list of company blogs that talk about Iceberg. The blogs are ordered from most recent to oldest.
 
 <!-- markdown-link-check-disable-next-line -->
+
```

Comment:
please remove this line so that `<!-- markdown-link-check-disable-next-line -->` is right above L27